web/js/map-OpenLayers.js

```diff
@@ -121,7 +113,8 @@ var fixmystreet = fixmystreet || {};
         return fixmystreet.markers.getFeaturesByAttribute('id', problem_id)[0];
       },
 
-      marker_size_for_zoom: function(zoom) {
+      marker_size_for_zoom: function() {
```

Comment:
Could this function (and related) take an argument, and then default to what you have in none is provided? Then the function names still make sense :)